static/app/views/dashboardsV2/gridLayout/sortableWidget.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+import IssueWidgetCard from '../issueWidgetCard';
+import {Widget, WidgetType} from '../types';
+
+import WidgetCard from './widgetCard';
+
+type Props = {
+  widget: Widget;
+  dragId: string;
+  isEditing: boolean;
+  onDelete: () => void;
+  onEdit: () => void;
+};
+
+function SortableWidget(props: Props) {
+  const {widget, isEditing, onDelete, onEdit} = props;
```

Comment:
You can inline this into the function params

```suggestion
function SortableWidget({widget, isEditing, onDelete, onEdit}: Props) {
```